CompetitionTemplate/src/competition/subsystems/autonomous/selection/SetupTraversePortcullisCommand.java

```diff
@@ -0,0 +1,40 @@
+package competition.subsystems.autonomous.selection;
+
+import competition.subsystems.autonomous.RaiseArmAndTraverseDefenseCommandGroup;
+import competition.subsystems.autonomous.TraversePortcullisCommandGroup;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SetupTraversePortcullisCommand extends BaseAutonomousModeSetCommand{
+    public TraversePortcullisCommandGroup auto;
+    public RaiseArmAndTraverseDefenseCommandGroup raiseArm;
+    
+    DoubleProperty traversePortcullisPower;
+    DoubleProperty traversePortcullisHeading;
+    DoubleProperty traversePortcullisMin;
+    DoubleProperty traversePortcullisMax;
+    
+    public SetupTraversePortcullisCommand(AutonomousModeSelector autonomousModeSelector, 
+            XPropertyManager propManager, 
+            RaiseArmAndTraverseDefenseCommandGroup raiseArm) {
+        super(autonomousModeSelector);
+        traversePortcullisPower = propManager.createPersistentProperty("powerForTraversingPortcullis", 0.5);
```

Comment:
I'd make these look the same as the other setup auto programs so the format looks consistent